src/game-views/Sidebar.css

```diff
@@ -1,17 +1,29 @@
 .Sidebar {
     background: #ddd;
-    border-left: 3px solid #282c34;
     display: flex;
     flex-direction: column;
     grid-area: sidebar;
+    overflow-x: hidden;
```

Comment:
🤔 
are you really just saying `overflow: hidden` and not just x?

Reply:
and also what is this doing if we also have `overflow: hidden` on `SidebarPlayer-contents`?